pkg/components/frontend.go

```diff
@@ -138,3 +139,34 @@ func (f *frontend) IsRunning(_ context.Context) bool {
 	}
 	return true
 }
+
+func (f *Frontend) PrintDashboardLog() {
```

Comment:
Suggestion: I think it's better to add a util function, for example, `HostIP()` and use  in the logging:

```go
...
// get the port from c.config.Cluster.Frontend.HTTPAddr
port := getPort(c.config.Cluster.Frontend.HTTPAddr)

c.logger.V(0).Infof("To view dashboard by accessing: %s", logger.Bold(fmt.Sprintf("http://%s:%d", HostIP(), port)))
...
```